variant-inc/lazy-helm-charts
# Feature/cloud 1408 whitelist auth policy

# Description

Merging to 901 branch, then 901 will have a PR for master.

From outside cluster, require token if authentication is true
From inside cluster, allow /health /metrics as default and more in values.yaml


Fixes [#1437](https://drivevariant.atlassian.net/browse/CLOUD-1437)
Feature [#1408](https://drivevariant.atlassian.net/browse/CLOUD-1408)

## Type of change


## How Has This Been Tested?
  - See Github workflow: https://github.com/variant-inc/lazy-helm-charts/actions/runs/1825257803
  - Run `helm unittest -3 --strict -f ci/unit/jwt-whitelist.yaml ./charts/*` in root dir of branch
  - See Octopus Deploy: https://octopus.apps.ops-drivevariant.com/app#/Spaces-2/projects/lazy-helm-charts/deployments/releases/0.1.0-cloud-1408-white0001.66/deployments/Deployments-29224?activeTab=taskSummary
  - Run `ct install --charts charts/variant-api/ --namespace lazy-helm-charts` in root dir of branch

## Checklist
